website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -67,7 +67,7 @@ Use the Alert to communicate validation errors. For more details, refer to the [
     <A.Description>Correct the formatting of the following fields to update your user profile:
     </A.Description>
     <A.Description>
-    <Hds::Link::Inline @href="...">Email address</Hds::Link::Inline>
+    <Hds::Link::Inline @href="..." @color="secondary">Email address</Hds::Link::Inline>
```

Comment:
This seemed to be what we would want to use here but please let me know if this isn't correct.

Reply:
The other examples in the Web docs should be reviewed to ensure they use the desired link colors. We should also consider adding additional documentation clarifying our recommendation for links used in the Alert Description.

Reply:
For example, should this tertiary button, which visually looks like a primary link, be using the blue color? https://hds-website-git-hds-2396-fix-alert-links-hashicorp.vercel.app/components/alert#buttons

And is there other clarification we could add about usage of secondary vs. primary links included in the "Description"? @heatherlarsen @jorytindall

Reply:
> For example, should this tertiary button, which visually looks like a primary link, be using the blue color?

Yes, the tertiary button does not come in a neutral color. 

> And is there other clarification we could add about usage of secondary vs. primary links included in the "Description"?

I thought we agreed that this could use more clarity. I feel like this is a change you can make at the same time in this PR, but if you'd rather one of us make this change later, please create a Jira task for that.

Reply:
Thanks, I can see if I could add some updated language but wanted to tag you and Jory in case you had specific ideas since it's a design-related concern.

Reply:
Actually in looking at this I'm confused about how to change it:

<img width="897" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/49d2fc61-b07f-42b1-ba4b-9ac6ab5f2917">

In the code example we include an inline link as an example of rich content in the description. Should I mention that inline links are ok to include in the description and we recommend using secondary color for those too?

I could leave updates to docs for a separate PR, but I worry about releasing a change like this and not updating docs at the same time could cause confusion... What do you think though?

Reply:
May be that this question has been left unanswered?